server/cluster.go

```diff
@@ -148,6 +148,10 @@ func encodeRegionSearchKey(endKey []byte) string {
 	return string(append([]byte{'z'}, endKey...))
 }
 
+func encodeRegionStartKey(startKey []byte) string {
```

Comment:
Can we reuse encodeRegionSearchKey function?
Change `endKey` variable to `key`?